tests/upgrades-images-signed/upgrade_test.go

```diff
@@ -15,33 +16,29 @@ var _ = Describe("cOS Upgrade tests - Images signed", func() {
 	})
 
 	AfterEach(func() {
-		// Try to gather mtree logs on failure
 		if CurrentGinkgoTestDescription().Failed {
 			s.GatherAllLogs()
 		}
-		if CurrentGinkgoTestDescription().Failed == false {
-			s.Reset()
-		}
 	})
 	Context("After install", func() {
 		When("upgrading", func() {
-			It("upgrades to latest available (master) and reset", func() {
-				out, err := s.Command("cos-upgrade")
```

Comment:
yesp, Im planning to extract that upgrade from master path into a new test and compare the actual time it takes by itself.

Just by skipping the reset after each test we gain time but...we may lose time in teh x86_64 tests :/

Reply:
good point!

Reply:
x86_64 seem to reduce the time from 11 minutes to about 6/7. But, if we need another job for the cos-upgrade that means another worker (+2 as we run both squash/nonsquash) which I dont think we can afford to have as it would just slow down the whole CI.

Gonna have a look at the focus stuff, because we may be able to just run all the upgrade jobs together in x86 while separating them in arm64, if ginkgo allows for multiple tags/focus groups (or maybe I need to bump it to ginkgo v2 for that :P)